console/console-timing-logging-manual.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Console Timing Methods - Logging Manual Test</title>
+<meta name="author" title="Dominic Farolino" href="mailto:domfarolino@gmail.com">
+<meta name="assert" content="Console timing methods">
+<link rel="help" href="https://console.spec.whatwg.org/#timing">
+</head>
+<body>
+<p>Open the console inside the developer tools. It should contain entries whose contents are:</p>
+<p><code>default: &lt;some time></code></p>
+<p><code>default: &lt;some time></code></p>
+<p><code>default: &lt;some time></code></p>
+<p><code>default: &lt;some time> extra data</code></p>
+<p><code>default: &lt;some time></code></p>
+<p><code>default: &lt;some time></code></p>
+<p><code>default: &lt;some time> extra data</code></p>
+<p><code>default: &lt;some time></code></p>
+<p><code>default: &lt;some time></code></p>
+<p><code>default: &lt;some time> extra data</code></p>
+<p><code>default: &lt;some time></code></p>
+<p><code>a label: &lt;some time></code></p>
+<p><code>a label: &lt;some time> extra data</code></p>
+<p><code>a label: &lt;some time></code></p>
+<p style="color:grey;">[some warning message indicating that a timer for label "b" does not exist]</p>
+
+<script>
+console.time();
+console.timeLog();
+console.timeEnd();
+
+console.time(undefined);
+console.timeLog(undefined);
+console.timeLog(undefined, "extra data");
+console.timeEnd(undefined);
+
+console.time("default");
+console.timeLog("default");
+console.timeLog("default", "extra data");
+console.timeEnd("default");
+
+console.time({toString() {return "default"}});
+console.timeLog({toString() {return "default"}});
+console.timeLog({toString() {return "default"}}, "extra data");
+console.timeEnd({toString() {return "default"}});
```

Comment:
It'd be good to test toString() conversion with a non-default label as well, to guard against implementations that fall back to "default" if typeof !== string